### PR TITLE
feat: implemented JsonSchema for FixedPoint

### PR DESCRIPTION
**Pull request by antonivakin**
Implemented schemars::JsonSchema for FixedPoint

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -15,6 +15,7 @@
 //! - `parity` — [`parity-scale-codec`][parity_scale_codec] support (`Encode` and `Decode`
 //!   implementations).
 //! - `serde` — support for `serde`. Enabled by default.
+//! - `schemars` — support for `schemars`. Enabled by default.
```
Comment (loyd) [inline src/lib.rs]: It's disabled by default. Hmm, `serde` is also disabled by default.

File: src/lib.rs [hunk 2]
```diff
@@ -165,6 +166,10 @@ pub mod ops;
 #[cfg_attr(docsrs, doc(cfg(feature = "serde")))]
 pub mod serde;
 
+#[cfg(feature = "schemars")]
+#[cfg_attr(docsrs, doc(cfg(feature = "schemars")))]
+pub mod schemars;
```
Comment (loyd) [inline src/lib.rs]: I believe we shouldn't expose any parts of `schemars` out of this crate.

File: src/schemars.rs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+use crate::{FixedPoint};
+
+impl<I, P> JsonSchema for FixedPoint<I, P> {
+    fn is_referenceable() -> bool {
+        false
+    }
+
+    fn schema_name() -> String {
+        "String".to_owned()
```
Comment (loyd) [inline src/schemars.rs]: Why not `FixedPoint` here?

According to the [docs](https://docs.rs/schemars/latest/schemars/trait.JsonSchema.html#tymethod.schema_name):
> The name of the generated JSON Schema.
> This is used as the title for root schemas, and the key within the root’s definitions property for subschemas.

Comment (loyd) [reply src/schemars.rs]: Ideally, it should be `FixedPoint(<i>, <p>)`, but I'm not sure whether it's acceptable by all generators or not, so `FixedPoint` is good enough for now.

File: src/lib.rs (hunk 1 above)
Comment (antonivakin) [reply src/lib.rs]: Edited comment

File: src/lib.rs (hunk 2 above)
Comment (antonivakin) [reply src/lib.rs]: fixed

File: src/schemars.rs (hunk 3 above)
Comment (antonivakin) [reply src/schemars.rs]: renamed

**Pull request merged**